website/docs/d/advanced_clusters.html.markdown

```diff
@@ -42,9 +42,65 @@ data "mongodbatlas_clusters" "example" {
 }
 ```
 
+Example using latest schema with independent (asymmetric) shards in the cluster:
```

Comment:
same comments made to singular data source apply for the plural, we can adjust once we agree on changes in the singular

Reply:
applied all changes to both data sources